### PR TITLE
0.11 release notes

### PR DESCRIPTION
**Pull request by stefanv**

### CODE REVIEW

File: doc/release/release_0.11.txt [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+- Region Adjacency Graphs
+  - Color distance RAGs (#1031)
+  - Threshold Cut on RAGs (#1031)
+  - Similarity RAGs (#1080)
+  - Normalized Cut on RAGs (#1080)
+  - RAG drawing (#1087)
+  - Hierarchical merging (#1100)
+- Sub-pixel shift registration (#1066)
+- NL-means denoising (#874)
```
Comment (jni) [inline doc/release/release_0.11.txt]: non-local means?

File: doc/release/release_0.11.txt [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+- Sliding window histogram (#1127)
+- More illuminan ts in color conversion (#1130)
```
Comment (jni) [inline doc/release/release_0.11.txt]: illuminants (typo)

File: doc/release/release_0.11.txt [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+For more information, examples, and documentation, please visit our website:
+
+http://scikit-image.org
+
+Highlights
+----------
+For this release, we merged over 200 pull requests with bug fixes,
+cleanups, improved documentation and new features.  Highlights
+include:
```
Comment (jni) [inline doc/release/release_0.11.txt]: I would also add the massively overhauled IO module!

**Comment by jni** [comment]
This is incredibly exciting! =) Just made a couple of minor comments!

**Comment by stefanv** [comment]
How does that look?

**Comment by ahojnnes** [comment]
Looks good!

**Comment by emmanuelle** [comment]
looks good to me!

File: doc/release/release_0.10.txt [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+- Almar Klein
+- Jeremy Metz
+- Juan Nunez-Iglesias
+- François Orieux
+- Guillem Palou
+- Rishabh Raj
+- Thomas Robitaille
+- Michal Romaniuk
+- Johannes L. Schönberger
+- Steven Sylvester
```
Comment (blink1073) [inline doc/release/release_0.10.txt]: Silvester*

**Comment by blink1073** [comment]
Looks good other than spelling the janitor's name wrong  :smile:

File: doc/release/release_0.11.txt [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+- Improve conditioning of geometric transformations (#1319)
+- Standardize handling of multi-image files (#1200)
+- Ellipse structuring element (#1298)
+- Multi-line drawing tool (#1065), line handle style (#1179)
+- Point in polygon testing (#1123)
+- Rotation around a specified center (#1168)
+- Add `shape` option to drawing functions (#1222)
+- Faster regionprops (#1351)
+- `skimage.future` package (#1365)
+- More robust I/O module
```
Comment (sciunto) [inline doc/release/release_0.11.txt]: missing IDs?

**Comment by stefanv** [comment]
Woops, that was in the old release notes (0.11 I did it correctly).

File: doc/release/release_0.11.txt (hunk 5 above)
Comment (stefanv) [inline doc/release/release_0.11.txt]: I don't think this happened in any specific PR.  It's more like Steven
waved his magic wand over a couple of PRs.  If you can identify any I
should use, please let me know!

File: doc/release/release_0.10.txt [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+- Pietro Berkes
+- Vighnesh Birodkar
+- François Boulogne
+- Olivier Debeir
+- Christoph Deil
+- Jaidev Deshpande
+- Jaime Frio
+- Jostein Bø Fløystad
+- Neeraj Gangwar
+- Christopher Gohlke
```
Comment (cgohlke) [inline doc/release/release_0.10.txt]: Christoph Gohlke

File: doc/release/release_0.10.txt [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+- Michal Romaniuk
+- Johannes L. Schönberger
+- Steven Sylvester
+- Julian Taylor
+- Gregor Thalhammer
+- Matthew Trentacoste
+- Siva Prasad Varma
+- Guillem Palou Visa
+- Stefan van der Walt
+- Josh Warner
```
Comment (JDWarner) [inline doc/release/release_0.10.txt]: For usual correspondence I use Josh, but prefer Joshua for publications & professional use.

File: doc/release/release_0.11.txt (hunk 5 above)
Comment (blink1073) [inline doc/release/release_0.11.txt]: #1189 is a good candidate.

**Comment by jni** [comment]
What's Travis's problem? It's reproducible... =\

**Comment by jni** [comment]
I stand corrected about Travis, but there's still a few corrections needed here!

**Pull request merged**